wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -1,18 +1,26 @@
 package org.bitcoins.wallet
```

Comment:
Everything in this file (`Wallet.scala`) is important to review, this is the main implementation of the wallet dlcs